package.json

```diff
@@ -35,7 +38,7 @@
     "@vscode/test-electron": "2.3.0",
     "@vscode/vsce": "2.21.1",
     "acorn": "8.8.2",
-    "ajv": "6.12.6",
+    "ajv": "8.17.1",
```

Comment:
@madhur310 we need to check other packages to see if bumping ajv to current version. Unit tests are on the light side across the repo